microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -148,7 +148,7 @@ def _send_community_test_msg(
         test_status: TestStatus,
     ) -> None:
         community_msg = create_test_result_message(
```

Comment:
change those names too, search "community" to try best to change names.

Reply:
The reason I skipped changing these is because these actually are "community" tests. Our reason for updating from community to subtest is to accommodate any test framework which follow a similar pattern in future, and doesn't fall in the category of "community" test. It's the similar case for `xfs` and `ltp` too. Please let me know if you think otherwise

Reply:
The "community" is the original concept, which is replaced by "sub tests". So, it doesn't matter it's community test or not. The reason to send subtests message, is we won't dump test results for each "subtests". Yes, it is community tests, and it's also sub tests. We don't care about whether it's community tests now. Less concept, more clear.